packages/ui-map-components/src/components/LeafletMap.tsx

```diff
@@ -217,9 +221,7 @@ export class LeafletMap extends Component<LeafletMapProps> {
 
   flyToBounds = (bounds: LatLngBoundsExpression) => {
     if (!areBoundsValid(bounds)) return;
-    this.map?.fitBounds(bounds, {
-      animate: true,
-    });
+    this.map?.fitBounds(bounds, { animate: true });
```

Comment:
Did you end up finding a nice animation for this one?

Reply:
I did get it working pretty nicely with map.fitToBounds and it while it was nice and smooth it was much less snappy and a very different feel from how it currently works. So I just went for feature parity and added a note to [the list ](https://beyond-essential.slab.com/posts/web-frontend-rewrite-03l47wk9#h18bd-dev-testing-polishing) of polish as a reminder to look at it later with Edwin.